gtda/plotting/diagrams.py

```diff
@@ -5,9 +5,103 @@
 
 import numpy as np
 import plotly.graph_objs as gobj
+from ._plot import _plot_image
 
 
-def plot_betti_curves(betti_curves, homology_dimensions=None, samplings=None):
+def plot_heat_kernel(heat_kernel, samplings, homology_dimension=0):
+    """Plot the heat kernel of a single persistence diagram in a single dimension.
+
+    Parameters
+    ----------
+    heat_kernel : ndarray of shape (homology_dimensions, n_bins, \
+        n_bins)
+        Raster image: one image per homology dimension. Index i along axis 0
+        corresponds to the i-th homology dimension in
+        :attr:`homology_dimensions_`.
+
+    homology_dimension : int or None, default: ``0``
+        Homology dimension for which the heat kernel should be plotted.
+
+    samplings : ndarray of shape (n_homology_dimension, n_values), \
```

Comment:
Shouldn't `n_values` be `n_bins`? Do we have a similar issue in `HeatKernel`?

Reply:
n_homology_dimensions (plural)

Reply:
yes, changed